mpmath/functions/zetazeros.py

```diff
@@ -121,7 +121,7 @@ def separate_zeros_in_block(ctx, zero_number_block, T, V, limitloop=None,
                 f = lambda x: ctx.rs_z(x,derivative=1)
                 t0=T[kMax-1]
                 t1 = T[kMax]
-                t=ctx.findroot(f,  (t0,t1), solver ='illinois',verify=False, verbose=False)
+                t=ctx.findroot(f,  (t0,t1), solver ='anderson',verify=False, verbose=False)
```

Comment:
Not sure why `verify=False` is used here.

Reply:
I don't think it has any effect, but I left it to avoid an unrelated change.

Reply:
Also added to https://github.com/mpmath/mpmath/pull/961